app/app.py

```diff
@@ -5,20 +5,22 @@
 
 app = Flask(__name__)
 
+CORS_HEADERS = {"Access-Control-Allow-Origin": "*"}
+
+
 @app.route(API_VERSION + "/question/<question_type>/<question_id>", methods=["GET"])
 def question_router(question_type, question_id):
-
     seed = int(request.args.get("seed", time()))
     question, status = get_question(question_type, question_id, seed)
 
-    return question.json, status
+    return question.json, status, CORS_HEADERS
```

Comment:
Like I mentioned in another PR we will need to have a think about returning proper HttpResponse objects soon because this approach will get messy

Reply:
agree